Description: install libs to /usr/lib/<multiarch>.
Drop extra /libgig subdir
Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Forwarded: cuse@users.sourceforge.net

Index: libgig/akai.pc.in
===================================================================
--- libgig.orig/akai.pc.in
+++ libgig/akai.pc.in
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@/libgig
+libdir=@libdir@
 includedir=@includedir@/libgig
 
 Name: akai
Index: libgig/gig.pc.in
===================================================================
--- libgig.orig/gig.pc.in
+++ libgig/gig.pc.in
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@/libgig
+libdir=@libdir@
 includedir=@includedir@/libgig
 
 Name: gig
Index: libgig/libgig.spec.in
===================================================================
--- libgig.orig/libgig.spec.in
+++ libgig/libgig.spec.in
@@ -80,18 +80,18 @@ if [ -d $RPM_BUILD_ROOT ]; then rm -rf $
 %defattr(-,root,root)
 %doc AUTHORS COPYING ChangeLog NEWS README TODO
 %{prefix}/bin/*
-%{prefix}/lib/libgig/libgig.so*
-%{prefix}/lib/libgig/libakai.so*
+%{prefix}/lib/libgig.so*
+%{prefix}/lib/libakai.so*
 %{prefix}/share/man/man1/*
 
 %files devel
 %defattr(-,root,root)
 %doc doc/html/*
-%{prefix}/lib/libgig/libgig.a
-%{prefix}/lib/libgig/libgig.la
+%{prefix}/lib/libgig.a
+%{prefix}/lib/libgig.la
 %{prefix}/lib/pkgconfig/gig.pc
-%{prefix}/lib/libgig/libakai.a
-%{prefix}/lib/libgig/libakai.la
+%{prefix}/lib/libakai.a
+%{prefix}/lib/libakai.la
 %{prefix}/lib/pkgconfig/akai.pc
 %{prefix}/include/libgig/*
 
Index: libgig/src/Makefile.am
===================================================================
--- libgig.orig/src/Makefile.am
+++ libgig/src/Makefile.am
@@ -1,7 +1,7 @@
 # to prevent compile errors on some systems
 AM_CXXFLAGS = -pedantic -Wreturn-type
 
-pkglib_LTLIBRARIES = libgig.la libakai.la
+lib_LTLIBRARIES = libgig.la libakai.la
 
 libgigincludedir = $(includedir)/libgig
 libgiginclude_HEADERS = RIFF.h DLS.h SF.h gig.h Korg.h Serialization.h
